openfoodfacts_taxonomy_parser/tests/unit/test_parser_unit.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import re
+
+#importing parser
+import os,sys
+script_dir = os.path.dirname( '__file__' )
+mymodule_dir = os.path.join( script_dir, '..' , '..', 'parser')
+sys.path.append( mymodule_dir )
+import parser
```

Comment:
you should not need this if you add a __init__.py to parser directory…

Reply:
Also you renamed the wrong "parser" directory.

The one to name `openfoodfacts_taxonomy_parser` would be the one above `parser.py`

and then in you test you would do: `from openfoodfacts_taxonomy_parser import parser`

(sorry there again, my previous comment could have been misleading…)